pom.xml

```diff
@@ -100,7 +100,7 @@
     <module>connector/kafka-0-10-assembly</module>
     <module>connector/kafka-0-10-sql</module>
     <module>connector/avro</module>
-    <module>connect</module>
+    <module>connector/connect</module>
```

Comment:
So this will be a module compiled by default, and there will be no profile like `-Pconnect`, right? (I'm thinking about how to compile this module under Centos 6 or 7.)

Reply:
actually yeah, I copied it from avro but I think what you said is correct.